src/traces/scattermapbox/convert.js

```diff
@@ -61,7 +89,7 @@ module.exports = function convert(gd, calcTrace) {
         fill.layout.visibility = 'visible';
 
         Lib.extendFlat(fill.paint, {
-            'fill-color': trace.fillcolor
+            'fill-color': trace.fillcolor,
```

Comment:
Would you mind reverting any minor syntax changes & refactors like this one in this PR?
That way it would be easier to keep track of the actual changes required to implement the new feature.
Thank you!

Reply:
Also you are welcome to suggest adding new lint rules here: https://github.com/plotly/plotly.js/issues/3162